public/index.html

```diff
@@ -34,7 +34,7 @@
 <body class='flex'>
   <div class="content">
     <div class="title white-and-blue">
-      <h1 data-translation-id="page_title" class='h1'>Twin Cities Aid Map</h1>
+      <h1 data-translation-id="page_title" class='h1'>Twin Cities Mutual Aid Map</h1>
```

Comment:
I'm not sure whether we should change the title here, how does the longer title look on screens? Also, it diverges from the translations.